cmd/helm/plugin_test.go

```diff
@@ -277,11 +277,6 @@ func TestPluginDynamicCompletion(t *testing.T) {
 		cmd:    "__complete echo -n mynamespace ''",
 		golden: "output/plugin_echo_no_directive.txt",
 		rels:   []*release.Release{},
-	}, {
-		name:   "completion for plugin bad directive",
-		cmd:    "__complete echo ''",
-		golden: "output/plugin_echo_bad_directive.txt",
-		rels:   []*release.Release{},
```

Comment:
Out of curiosity, why was this test dropped?

Reply:
(From the description of the PR 😄 ):

A completion test was removed in this PR as it was failing with the new version of Cobra.
The test was actually testing Cobra's behaviour (which changed in the new release), which is not something Helm tests should focus on, so I simply removed it.

I don't expect Cobra's change of behaviour to have much of an impact on Helm, and I was being over-zealous when I wrote the test.